README.md

```diff
@@ -1,126 +1,100 @@
-Note that the priority is: `~/.box64rc` > `/etc/box64.box64rc` > command line
-So, your settings in `~/.box64rc` may override the setting from your command line...
+## 📄 Additional Platform-Specific Notes
 
-----
+### Unity Game Emulation
+- Many Unity games require OpenGL 3+, which may be challenging on ARM SBCs.
+- **Tip for Pi4 and Pi5 Users:** Set `MESA_GL_VERSION_OVERRIDE=3.2` with `BOX64_DYNAREC_STRONGMEM=1` to prevent freezes and enable strong memory mode.
+- **Using Panfrost:** For better compatibility on ARM, enable `PAN_MESA_DEBUG=gl3` to force higher OpenGL profiles. This can help if a game starts but quits unexpectedly before showing any content.
 
-Notes about Unity game emulation
-----
+Let me know if you’d like further adjustments!
 
-Running Unity games should just work, but you should also note that many Unity3D games require OpenGL 3+ which can be tricky to provide on ARM SBC (single-board computers). Also many newer Unity3D (like KSP) games use the BC7 compressed textures, which is not supported on many ARM integrated GPU.
-Hint: on Pi4 and Pi5 use `MESA_GL_VERSION_OVERRIDE=3.2`, with `BOX64_DYNAREC_STRONGMEM=1` to prevent freezes and enable strong memory mode, and use Panfrost `PAN_MESA_DEBUG=gl3` to use higher profile if the game starts then quits before showing anything.
+### GTK Programs
+- Box64 now wraps GTK libraries, supporting both gtk2 and gtk3.
 
-----
+### Steam
+- Steam requires Box86 due to its 32-bit client app, but uses 64-bit local server binaries. Systems with less than 6GB RAM may need a swap file for optimal performance.
 
-Notes about GTK programs
-----
+### Wine
+- Box64 supports Wine64 and Proton. For 32-bit components, Box86 is required. Systems with both Box64 and Box86 can run 32- and 64-bit Windows programs.
 
-GTK libraries are now wrapped on box64, both gtk2 and gtk3.
+### Vulkan
+- Box64 wraps Vulkan libraries. Limited testing done with AMD RX550 and Freedreno drivers.
 
-----
+---
 
-Notes about Steam
-----
+## ❤️ Acknowledgments
 
-Note that Steam is a hybrid 32-bit / 64-bit. You NEED box86 to run Steam, as the client app is a 32-bit binary. It also uses a 64-bit local server binaries, and that steamwebhelper process is now mandatory, even on the "small mode". And that process will eat lots of memory. So machine with less the 6Gb of RAM will need a swapfile to use Steam.
+A huge thank you to everyone who has contributed to Box64 development! Special thanks to:

```

Comment:
Please do not change the contribution part.

Reply:
Understood, I'm going to fix it